packages/flutter/test/material/slider_test.dart

```diff
@@ -622,104 +623,104 @@ void main() {
 
     await tester.pumpWidget(buildApp());
 
-    final RenderBox sliderBox = tester.firstRenderObject<RenderBox>(find.byType(Slider));
+    final MaterialInkController material = Material.of(tester.element(find.byType(Slider)));
```

Comment:
Why do the tests check the `MaterialInkController` instead of the `RenderBox` now?

Reply:
answered above